lib/src/Base/Algo/LeastSquaresMetaModelSelection.cxx

```diff
@@ -144,15 +145,18 @@ void LeastSquaresMetaModelSelection::run(const DesignProxy & proxy)
     }
     else
     {
-      if (!(error <= alpha * minimumError))
+      if (basisSequenceFactory_.getImplementation()->currentIndices_.getSize() > smallBasisSize)
       {
-        LOGINFO(OSS() << "Error=" << error << " larger than " << alpha << "*" << minimumError << "=" << alpha * minimumError);
-        break;
-      }
-      if (error > maximumError)
-      {
-        LOGINFO(OSS() << "Error=" << error << " larger than " << maximumError);
-        break;
+        if (!(error <= alpha * minimumError))
```

Comment:
Géraud Blatman proposes an additional test to detect overfitting (see the blue box in Figure 5.8 page 123 of https://theses.hal.science/tel-00440197). It is based on the detection of two consecutive inceases of the cross validation error estimate.